### ISSUE TITLE
Disabling webserver from command line

### ISSUE THREAD

**Issue by andrew-stevenson-sociomantic** [stricaud/faup#99]
I want to build without the webserver without modifying the src (i.e. removing `cmake-options/webserver`). It seems like this should be simple but my cmake knowledge is clearly lacking.

I tried adding `-DWEBSERVER=0 -DFAUP_WEBSERVER=0` to the cmake call but it seems the existence of `cmake-options/webserver` overrides that.

Sorry for the support request - if there is a better forum let me know!

**Comment by stricaud** [comment]
no this is fine to ask here! Sorry for the delayed answer, I should actually document this so I am not closing this ticket to make sure I do not forget. 
Remove the file in the source tree: cmake-options/webserver
And this should do it.

Thanks for this report, it helps me to improve usability!

**Comment by stricaud** [comment]
also, please update to latest commit as I fixed a bug that linked to function that could not exist as the webserver was not compiled.

**Comment by andrew-stevenson-sociomantic** [comment]
I was hoping there was a method that doesn't involve modifying the source on disk. I am hooking it into an automated build and removing the file makes git say there are unchecked in changes which I get a warning about. It's not tragic, just hoping for a command line option.

Will check out the latest commit, thanks! I noticed the build problems but I'm only building the library so I dodged them this time :-)
